packages/flutter/test/material/user_accounts_drawer_header_test.dart

```diff
@@ -528,17 +549,17 @@ void main() {
 
     expect(tester.getSemantics(find.text('B')), matchesSemantics(
       label: 'B',
-      size: const Size(48.0, 48.0),
+      size: const Size(40.0, 40.0),
```

Comment:
Shouldn't the size of the tap targets still be 48.0 by 48.0, here and below? I think that's the minimum required size for mobile right?

Reply:
https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/theme_data.dart#L93

Reply:
I changed the Semantic widget place, which made the semantic widget returns to its original size of 48.0.  But the size of the children  (otherAccountsPicturesSize )  still is 40.0 because of the surrounding padding. That is what was originally done (before this PR), are we sure that's the right behaviour/design/guideline :) ?